Test Failure: LoadTestWithFailoverEnabled.testFiveMinuteRun PersistenceException FFDC

Test Failure: com.ibm.ws.concurrent.persistent.fat.load.LoadTestWithFailoverEnabled.testFiveMinuteRun

```
testFiveMinuteRun:junit.framework.AssertionFailedError: 2020-02-09-07:12:11:201 [Unexpected FFDC reporting javax.persistence.PersistenceException was found (count = 1): /WebSphere/Misc/jazz_build/jbe_boss0219/jazz/buildsystem/buildengine/eclipse/build/dev/image/output/wlp/usr/servers/com.ibm.ws.concurrent.persistent.fat.loadtest/logs/ffdc/ffdc_20.02.09_07.08.54.0.log
>------Start of DE processing------ = [2/9/20 7:08:54:610 GMT]
>Exception = javax.persistence.PersistenceException
>Source = com.ibm.ws.concurrent.persistent.internal.InvokerTask
>probeid = 611
>Stack Dump = javax.persistence.PersistenceException: 
>Exception Description: Error binding to externally managed transaction
>Internal Exception: javax.transaction.RollbackException: Transaction rolled back
>at org.eclipse.persistence.internal.jpa.transaction.JTATransactionWrapper.registerIfRequired(JTATransactionWrapper.java:145)
>at org.eclipse.persistence.internal.jpa.EntityManagerImpl.setJTATransactionWrapper(EntityManagerImpl.java:2168)
>at org.eclipse.persistence.internal.jpa.EntityManagerImpl.detectTransactionWrapper(EntityManagerImpl.java:893)
>at org.eclipse.persistence.internal.jpa.EntityManagerImpl.initialize(EntityManagerImpl.java:416)
>at org.eclipse.persistence.internal.jpa.EntityManagerImpl.<init>(EntityManagerImpl.java:409)
>at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.createEntityManagerImpl(EntityManagerFactoryDelegate.java:324)
>at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManagerImpl(EntityManagerFactoryImpl.java:337)
>at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManager(EntityManagerFactoryImpl.java:303)
>at com.ibm.wsspi.persistence.internal.PersistenceServiceUnitImpl.createEntityManager(PersistenceServiceUnitImpl.java:96)
>at com.ibm.ws.concurrent.persistent.db.DatabaseTaskStore.persist(DatabaseTaskStore.java:1208)
>at com.ibm.ws.concurrent.persistent.internal.InvokerTask.run(InvokerTask.java:575)
 at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:522)
 at java.util.concurrent.FutureTask.run(FutureTask.java:277)
 at com.ibm.ws.threading.internal.ExecutorServiceImpl$RunnableWrapper.run(ExecutorServiceImpl.java:239)
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1160)
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
 at java.lang.Thread.run(Thread.java:812)
Caused by: javax.transaction.RollbackException: Transaction rolled back
 at com.ibm.tx.jta.impl.TransactionImpl.registerSynchronization(TransactionImpl.java:2285)
 at com.ibm.tx.jta.impl.TransactionImpl.registerSynchronization(TransactionImpl.java:2265)
 at org.eclipse.persistence.internal.jpa.transaction.JTATransactionWrapper.registerIfRequired(JTATransactionWrapper.java:136)
 ... 16 more
```

The possibility of transaction rollback due to timeout is already expected.  The test currently allows for the RollbackException.  It needs to also allow for the PersistenceException that might chain it.

--- a/dev/com.ibm.ws.concurrent.persistent_fat_loadtest/test-applications/schedtest/src/web/LoadTestServlet.java
+++ b/dev/com.ibm.ws.concurrent.persistent_fat_loadtest/test-applications/schedtest/src/web/LoadTestServlet.java
@@ -96,7 +96,8 @@ public class LoadTestServlet extends FATServlet {
      */
     @Test
     @AllowedFFDC({
-        "javax.transaction.RollbackException" // under load, task executions sometimes roll back due to the 4 second transaction timeout that is used by the test
+        "javax.transaction.RollbackException", // under load, task executions sometimes roll back due to the 4 second transaction timeout that is used by the test
+        "javax.persistence.PersistenceException" // can wrap the RollbackException
     })
     public void testShortRun() throws Exception {
         // Clear results to start.
@@ -141,7 +142,8 @@ public class LoadTestServlet extends FATServlet {
     @Mode(TestMode.FULL)
     @AllowedFFDC({
         "java.lang.Exception",
-        "javax.transaction.RollbackException" // under load, task executions sometimes roll back due to the 4 second transaction timeout that is used by the test
+        "javax.transaction.RollbackException", // under load, task executions sometimes roll back due to the 4 second transaction timeout that is used by the test
+        "javax.persistence.PersistenceException" // can wrap the RollbackException
     })
     public void testFiveMinuteRun() throws Exception {
         // Clear results to start.

